src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,85 @@
+.card {
+  box-sizing: border-box;
+  background-color: $white-color;
```

Comment:
The variable `$white-color` is used here. Ensure that this variable is defined in your SCSS files. If it's not defined, the SCSS compilation will fail.